content/android-privacy-policy.md

```diff
@@ -1,14 +1,20 @@
 # Syncthing-Android Privacy Policy
 
-The app syncthing-android does not collect any user information.
+Read this document to understand how the app Syncthing-Android uses data.
+
+---
+
+Syncthing-Android does not collect any user information.
 
 The app does not store any location data.
-It requires the permission to access the location in the background, to look up the currently active Wi-Fi's SSID.
-That information is used to enable syncing only on user-configurable Wi-Fi's.
-The app never actually looks up your location in the process, and thus doesn't use or store location data.
+The location permission is used only to access background location in order to check for an SSID of the currently active WiFi.
+The sole purpose of this information is to limit synchronisation to user-configured WiFi connections.
+The app itself never collects the location in the process, and as such it also does not use or save any location data.
 
-The app uses the camera solely to scan QR-codes to enter a device ID.
-Pictures are not stored in the process.
+The app does not store any camera data.
```

Comment:
camera data?

Reply:
Yeah, I just copied the sentence from the location part. It is not really necessary.